content/pages/legal.md

```diff
@@ -1,172 +1,161 @@
-Social Security][7]. 
+Once you have a court order for your name change,
+[the first place to visit is Social Security][form-ssa-name].
 
-If you're able to print, you can save time by completing page 5 of [Form SS-5][2]
-in advance. At this stage, it is possible to simultaneously [update your gender marker][1] 
-with them when submitting Form SS-5 (see below Gender Marker section for details).
-This must be done eventually if changing your gender marker, 
-and can be completed without providing any medical documentation. Checking 
-this box now may save you time later.
+When you update your name with Social Security, they will give you a
+letter indicating that you requested changes.
 
-When you update with Social Security they will give you a letter indicating that 
-you requested changes. This letter is important, so hold on to it!
+- This letter is important, so be sure to keep it someplace safe.
 
-24 hours after updating with Social Security you can update your driver's
-license. Most DMVs should be able to change your name. Be sure to bring your
-letter from Social Security in addition to your court order. While you're at
-the DMV, you should also update your car registration and title.
+24 hours after updating with Social Security, you can update your driver's
+license.
 
-Once you've updated your driver's license you should start updating all the
-other places that know your name. Some important places to notify:
+- Most DMVs should be able to change your name.
+- Be sure to bring your letter from Social Security in addition to your original court order.
+- You should also update your voter registration
+  information and car registration/title if applicable.
 
- - Your employer's HR department
- - Banks
- - Credit card companies
- - And many more (it's amazing how many people know your name!)
+Once you've got a new ID, you can start elsewhere. Some important places:
+
+- Your employer's HR department
+- Bank and credit card companies
+- Your car insurance company
+- Your health insurance company
+- Your school or university
+- Doctors and/or medical providers
+- Your local post office (setup mail forwarding!)
+- And many more (it's amazing how many people know your name).
 
 # Updating your gender marker
 
-Currently, having an updated birth certificate is the quickest way to change
-your gender marker on other legal docs (like your driver's license). If you're at 
-least on hormones and can get a letter from a doctor or other provider, this 
-process may be possible for you.
+Currently, having an updated birth certificate is the quickest way to change your gender marker on other legal documents.
 
-At this time, Alabama does not offer a nonbinary gender identification.
+- If you're at least on hormones and can get a letter from a doctor or other provider, this process may be possible.
+- As of January 2025, Alabama does not offer a nonbinary or third gender identification option.
```

Comment:
I'd hesitate to put that date in there. AL has never offered this option. It didn't change